frontend/src/modules/activity/config/filters/main.ts

```diff
@@ -14,3 +15,16 @@ export const activityFilters: Record<string, FilterConfig> = {
   platform,
   sentiment,
 };
+export const activitySearchFilter: SearchFilterConfig = {
+  placeholder: 'Search activities',
+  apiFilterRenderer(value: string): any[] {
+    return [
+      {
+        or: [
+          { title: { textContains: value } },
+          { body: { contains: [value] } },
```

Comment:
Should `body` also be `textContains` with string? At least that's how we are using it right now and it's a string value

Reply:
fixed